tests/appsec/api_security/test_api_security_rc.py

```diff
@@ -8,31 +8,23 @@
     scenarios,
     weblog,
     features,
-    bug,
-    context,
 )
-from utils.tools import logger
 from tests.appsec.api_security.utils import BaseAppsecApiSecurityRcTest
 
 
 def get_schema(request, address):
     """Get api security schema from spans"""
     for _, _, span in interfaces.library.get_spans(request):
         meta = span.get("meta", {})
-        key = "_dd.appsec.s." + address
-        payload = meta.get(key)
```

Comment:
Could you keep this logs, it helps debugging

Reply:
Done